torch/_decomp/__init__.py

```diff
@@ -210,6 +210,7 @@ def register(op):
 def get_decompositions(
     aten_ops: Sequence[Union[torch._ops.OperatorBase, OpOverloadPacket]],
     type: str = "post_autograd",
+    ignore_cia: bool = False,
```

Comment:
it feels a bit weird that the export lazy dict logic needs to leak into this function. Why do we need it? (vs. just having the lazy dict get all ATen CIA ops the first time)